contracts/verifiers/ZKPVerifierBase.sol

```diff
@@ -312,4 +301,15 @@ abstract contract ZKPVerifierBase is IZKPVerifier, ContextUpgradeable {
     function getStateAddress() public view virtual returns (address) {
         return address(_getZKPVerifierStorage()._state);
     }
+
+    /// @dev Update a ZKP request
+    /// @param requestId The ID of the ZKP request
+    /// @param request The ZKP request data
```

Comment:
FYI
"The Solidity compiler only interprets tags if they are external or public. You are welcome to use similar comments for your internal and private functions, but those will not be parsed.

This may change in the future."